src/lib/tradingView/dydxfeed/index.ts

```diff
@@ -69,18 +66,9 @@ export const getDydxDatafeed = (
     onSymbolResolvedCallback: ResolveCallback,
     onResolveErrorCallback: ErrorCallback
   ) => {
-    const marketIds = getMarketIds(store.getState());
-    const symbols = getAllSymbols(marketIds);
-    const symbolItem = symbols.find(({ symbol }: any) => symbol === symbolName);
-
-    if (!symbolItem) {
-      onResolveErrorCallback('cannot resolve symbol');
-      return;
-    }
-
-    const { tickSizeDecimals } = getMarketConfig(symbolItem.symbol)(store.getState()) || {};
-
-    const pricescale = tickSizeDecimals ? 10 ** tickSizeDecimals : 100;
+    const marketId = getCurrentMarketId(store.getState());
+    const symbolItem = getSymbol(marketId || DEFAULT_MARKETID);
+    const pricescale = 100;
```

Comment:
w/o `tickSizeDecimals`, won't the markets with a lot of decimals be represented as `0.00` in a lot of cases?